main.go

```diff
@@ -67,6 +61,23 @@ func checkError(e error) {
 	}
 }
 
+// Load the beacon log from file
+func loadLog() BeaconLog {
+	beaconLogData := new(BeaconLog)
+
+	// read JSON file from disk
+	beaconLogFile, err := ioutil.ReadFile("./beacon_log.json")
+
+	checkError(err)
+
+	// unmarshal json and store it in the pointer to beaconLogData {?}
+	// NOTE: figure out if you can use `checkError` here; don't yet understand golang's idiomatic errors handling.
```

Comment:
Good to know. Let's make it a point of the refactor to eventually scan the project for errors and make for one consistent style.